bin/Makefile.base

```diff
@@ -15,6 +15,12 @@ test_fmt:
 	out=`gofmt -l -s .`; echo "$$out"; [ -z "$$out" ]
 	go vet ./...
 
+.PHONY: test_imports
+test_imports:
+	@echo Checking correct formatting and import lines of files
+	out=`goimports -l .`; echo "$$out"; [ -z "$$out" ]
```

Comment:
`goimports` is not available by default.

You can use something like the recipe used in `$(GOPATH)/bin/golint`, and reference the newly installed cmd using `$<`.

Reply:
Thanks yes indeed.